back/app/services/project_copy_service.rb

```diff
@@ -277,8 +278,8 @@ def yml_phases(shift_timestamps: 0, timeline_start_at: nil)
         'project_ref' => lookup_ref(phase.project_id, :project),
         'title_multiloc' => phase.title_multiloc,
         'description_multiloc' => phase.description_multiloc,
-        'start_at' => shift_timestamp(phase.start_at, shift_timestamps)&.iso8601,
-        'end_at' => shift_timestamp(phase.end_at, shift_timestamps)&.iso8601,
+        'start_at' => (phase.start_at + shift_timestamps.days)&.iso8601,
+        'end_at' => (phase.end_at + shift_timestamps.days)&.iso8601,
```

Comment:
We still need to offset eh the phase start & end dates using the `shift_timestamps` var, so I replaced the call to the `shift_timestamp` method with the logic of the method itself here, since otherwise we would conditionally skip the logic in the method and return `nil` for these dates.

@alexander-cit Can you think of a more elegant way to do this?

Reply:
The good news is we have a test that will catch if this change is reverted.

Reply:
```ruby
  def shift_timestamp(value, shift_timestamps, leave_blank: @local_copy)
    return if leave_blank

    value && (value + shift_timestamps.days)
  end
  
        'start_at' => shift_timestamp(phase.start_at, shift_timestamps, leave_blank: false)&.iso8601,
        'end_at' => shift_timestamp(phase.end_at, shift_timestamps, leave_blank: false)&.iso8601,

Reply:
Ooh, I like that. Thank you!